src/handlers/metadataDelta/handlers/addVerifiedMembers.ts

```diff
@@ -0,0 +1,70 @@
+import { mutate, query } from '~amplifyClient';
+import {
+  ColonyActionType,
+  CreateVerifiedMemberDocument,
+  CreateVerifiedMemberMutation,
+  CreateVerifiedMemberMutationVariables,
+  GetVerifiedMemberDocument,
+  GetVerifiedMemberQuery,
+  GetVerifiedMemberQueryVariables,
+} from '~graphql';
+import { ContractEvent } from '~types';
+import { verbose, writeActionFromEvent } from '~utils';
+import {
+  AddVerifiedMembersOperation,
+  MetadataDeltaOperationType,
+  MetadataDeltaOperation,
+} from '../types';
+
+const isAddVerifiedMembersOperation = (
+  operation: MetadataDeltaOperation,
+): operation is AddVerifiedMembersOperation => {
+  return (
+    operation.type === MetadataDeltaOperationType.ADD_VERIFIED_MEMBERS &&
+    operation.payload !== undefined &&
+    Array.isArray(operation.payload)
```

Comment:
You do the type check in `parseOperation`, so the individual handlers only need to validate the payload, not the entire operation hopefully?